src/kevedit/misc.c

```diff
@@ -54,6 +54,53 @@
 #include "zlaunch/dosbox.h"
 #endif
 
+static char * save_backup_begin(char * filename) {
+	char * bkp_filename;
+	int pos;
+
+	if (filename[0] == '\0') {
+		return str_dup(filename);
+	}
+
+#ifdef DOS
+	/* Do not assume LFN on DOS. */
+	bkp_filename = str_dup(filename);
+	bkp_filename[0] = '~';
```

Comment:
This blows up on dos because `filename` is actually a file *path* here, so `./untitled.zzt` becomes `~/untitled.zzt`

Reply:
Should be fixed, now. Sorry for taking a while!